va/proto/va.proto

```diff
@@ -16,13 +16,9 @@ service CAA {
 message IsCAAValidRequest {
   // Next unused field number: 6
   //
-  // TODO(#8023): dnsNames are being deprecated in favour of identifiers. Here,
-  // this is purely for consistency, because only DNS identifiers support CAA
-  // checks.
-  //
   // NOTE: Domain (or the identifier's value) may be a name with a wildcard
   // prefix (e.g. `*.example.com`)
```

Comment:
This comment should be moved above `Identifier` and should specify something more like "for DNS identifiers, this may contain a wildcard domain name (e.g. `*.example.com`)"